avm/res/network/virtual-network-gateway/main.json

```
@@ -1246,8 +1246,8 @@
           "metadata": {
             "_generator": {
               "name": "bicep",
-              "version": "0.23.1.45101",
-              "templateHash": "3164103777824557279"
+              "version": "0.24.24.22086",
+              "templateHash": "16896615660479182847"
```

```suggestion
              "version": "0.23.1.45101",
              "templateHash": "3164103777824557279"
```
Restoring as no new version needs to be published